test/lib/ufe/testSelectionNotification.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python
+
+#
+# Copyright 2020 Autodesk
```

Comment:
Not a big deal but since this is a new file, shouldn't the copyright year be 2023? If you do change it, don't bother re-running the preflight I'll merge it.

Reply:
Done updating copyright to 2023